shopfloor/services/checkout.py

```diff
@@ -478,7 +478,7 @@ def _select_lines_from_package(self, picking, selection_lines, package, **kw):
                     ),
                 },
             )
-        self._select_lines(lines)
+        self._select_lines(lines, prefill_qty=1)
```

Comment:
When scanning a source package, we don't increase the quantity
```suggestion
        self._select_lines(lines, prefill_qty=0)
```